src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -435,13 +425,13 @@
 
         "toolFingerprint": 
         {
-          "description": "A string that baselining mechanisms can merge with other data to help uniquely identify this issue, run-over-run.",
+          "description": "A string that contributes to the process of uniquely identifying the result.",
```

Comment:
A string that can help uniquely identify a result? is 'help' enough to indicate that the string doesn't itself uniquely identify a result?